src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -741,6 +792,24 @@ public class foo<T>
         }
 
         [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task InvokeWithOpenAngleCommitSeeOnSpace()
+        {
+            var markupBeforeCommit = @"class c
+{
+        /// <$$
+        void foo() { }
+}";
+
+            var expectedCodeAfterCommit = @"class c
+{
+        /// <see$$/>
+        void foo() { }
+}";
+
+            await VerifyCustomCommitProviderAsync(markupBeforeCommit, "see", expectedCodeAfterCommit, commitChar: ' ');
```

Comment:
Looking over this again, I seem to remember something about not doing "commit" tests like this. Would that be the case here as well? If so, can you remind me where/how they should be done?